### PR TITLE
prodyoGEEKY blog

### PR DESCRIPTION
**Pull request by nikita0511**
Team prodyoGEEKY's first blog! :)

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,26 +1,26 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (4.2.6)
+    activesupport (5.1.3)
```
Comment (v-thakkar) [inline Gemfile.lock]: Hi, thanks for sending the PR. :) If you've changed Gemfile.lock on purpose then please submit these changes as a separate PR.

File: blog/_posts/2017-08-07-two-geeks-and-a-summer-of-code.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+**Nick-** A CSS nerd, as he likes to call himself, Nick is a developer and UX/UI expert at Village Reach. He is excited about tools and capacity building. 
+
+**Tenly-** She works towards building a vibrant OpenLMIS community interacting with the existing and new OpenLMIS partner. She has an extensive experience of over eight years in international development in Africa. 
+
+**Karan-** A technology enthusiast, Karan loves to get his hands dirty with any new tool he comes across and has won numerous hackathons. One the most creative and sincere guys we know from our college, Karan has been a constant support to us and has helped us incredibly during our application too!
+
+**Chhavi-** A Noogler, Chhavi has participated as a mentor in various programs that support women in tech. She has also been facilitated by the Honorable President of India for securing first prize in Code for India.
+
+**Jigyasa-** She has recently been awarded Red Hat Women in Open Source Award. Jigyasa has carved her name as an Open Source enthusiast helping her friends as a mentor. She is also the director of Women who code.
```
Comment (v-thakkar) [inline blog/_posts/2017-08-07-two-geeks-and-a-summer-of-code.md]: AFAIK, Jigyasa is a regional WWC director. Change it to Women who code -> Women who code, Delhi. :)

**Review by v-thakkar — CHANGES_REQUESTED** [review]

File: blog/_posts/2017-08-07-two-geeks-and-a-summer-of-code.md (hunk 2 above)
Comment (prot95) [reply blog/_posts/2017-08-07-two-geeks-and-a-summer-of-code.md]: Sure, will do that. Thank you

File: Gemfile.lock (hunk 1 above)
Comment (nikita0511) [reply Gemfile.lock]: Hi Vaishali, we haven't changed the gemfile. So we are clueless as to how it got changed. Can you tell us what changes need to be made?

Comment (v-thakkar) [reply Gemfile.lock]: I think this is because of the dependencies. But running 'bundle exec jekyll serve' should have worked for you to watch the output. Anyway, for this PR just remove this file from teh commit and send a PR with blog post file only.

**Review by v-thakkar — CHANGES_REQUESTED** [review]
Hi, you are doing the same thing again. You removed this file and then added it again. Please check while files you're committing before sending PR. Also, you've added extra empty folder 'rubyinstaller'. Please remove it as well. PR should consist of only one file = your blogpost file.

**Review by v-thakkar — CHANGES_REQUESTED** [review]
So now apparently you are removing Gemfile.lock from the project which will break the system then. :)
This is just a simple point. Send a PR with your blog post file only. Everything else will be handled automatically by github. Also, please check the files in commit before you send the PR. 'git add .' adds all the changed files. You can do 'git add <file_name>' .

**Comment by nikita0511** [comment]
Hi, the local repo had become quite messy. So i forked a separate branch and created a new PR. Please ignore this one. Apologies for so many issues. :)
